resources/init.groovy.d/global-args-security.groovy

```diff
@@ -124,6 +133,29 @@ Thread.start {
     }
 
 
+    // SonarQube plugin config, source: https://github.com/ridakk/jenkins/blob/master/groovy-scripts/setup-sonarqube-plugin.groovy
+    println "--> setting SonarQube plugin"
```

Comment:
move println under the below if. Otherwise during every restart we will see that our groovy trying to "setting Son..."